src/juliaexepath.ts

```diff
@@ -17,6 +15,7 @@ async function setNewJuliaExePath(newPath: string) {
 
     child_process.exec(`"${newPath}" --version`, (error, stdout, stderr) => {
         if (error) {
+            actualJuliaExePath = null
```

Comment:
This handles a situation where we currently can up in a state where it appears that a julia binary was found, but in reality it doesn't work.